tools/theme-preview.sh

```diff
@@ -126,6 +126,7 @@ _lp_hostname() {
 }
 _lp_connection() {
   lp_connection=ssh
+  SSH_CONNECTION="192.168.0.254 56314 192.168.0.1 22"
```

Comment:
This change was already added in 8ea315e. Either drop the last commit, or rebase to latest `master`.